data/transactions/keyreg_test.go

```diff
@@ -54,6 +62,10 @@ func (balances keyregTestBalances) ConsensusParams() config.ConsensusParams {
 	return config.Consensus[balances.version]
 }
 
+func (balances keyregTestBalances) Round() basics.Round {
+	return basics.Round(8675309)
```

Comment:
in the spirit for testing against large numbers, how's about a number that won't fit into uint32 ?

Reply:
sure, done.